extensions/libraries/redcore/joomla/layout/file.php

```diff
@@ -1,667 +0,0 @@
-<?php
```

Comment:
Check files I changed. We don't use any Joomla files there, we have our own files instead.